benchmark/accuracy.py

```
@@ -0,0 +1,115 @@
+    # ── aggregate ──────────────────────────────────────────────────────────
+    total = len(cases)
+    tp = fp = fn = tn = 0
+    per_tp = defaultdict(int)
+    per_fn = defaultdict(int)
+    per_fp = defaultdict(int)
+    wrong  = []
+
+    for utt, expected, predicted, conf in results:
+        if expected is not None:
+            if predicted == expected:
+                tp += 1; per_tp[expected] += 1
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)